lib/tstracker/chainbs.go

```diff
@@ -129,3 +132,47 @@ func NewTrackingPgChainstore(ctx context.Context, conn string) (TrackingChainsto
 
 	return &tcs{PgBlockstore: pgbs}, nil
 }
+
+func (*tcs) DeleteMany([]cid.Cid) error {
+	return errors.New("not supported")
+}
+
+func (*tcs) View(cid.Cid, func([]byte) error) error {
+	return errors.New("not supported")
+}
+
+func (t *tcs) Store() blockstore.Blockstore {
+	return t
+}
+func (t *tcs) Head(ctx context.Context) cid.Cid {
+	tsk, _, err := t.CurrentDbTipSetKey(ctx)
+	if err != nil {
+		return cid.Undef
+	}
+	cids := tsk.Cids()
+	if len(cids) > 0 {
+		return cids[0]
+	}
+	return cid.Undef
+}
+func (t *tcs) CidAtHeight(ctx context.Context, h int64) (cid.Cid, error) {
+	tsd, err := t.GetFilTipSetHead(ctx)
+	if err != nil {
+		return cid.Undef, err
+	}
+
+	tsd, err = t.FindFilTipSet(ctx, tsd.TipSetCids, abi.ChainEpoch(h))
```

Comment:
This is incorrect - you need to specify a walkback, something like ( untested )
```suggestion
	tsd, err = t.FindFilTipSet(ctx, tsd.TipSetCids, tsd.Epoch - abi.ChainEpoch(h))
```